AppKit/CPSearchField.j

```diff
@@ -915,16 +915,25 @@ var CPRecentsAutosaveNameKey            = @"CPRecentsAutosaveNameKey",
     var destination = [_info objectForKey:CPObservedObjectKey],
         keyPath     = [_info objectForKey:CPObservedKeyPathKey];
 
-    var formatString = _predicateFormat.replace(/\$value/g, "%@");
     [self suppressSpecificNotificationFromObject:destination keyPath:keyPath];
 
     if (aValue)
-        [_controller setFilterPredicate:[CPPredicate predicateWithFormat:formatString, aValue]];
+    {
+        var valueCount   = [[_predicateFormat componentsSeparatedByString:@"$value"] count] - 1,
+            formatString = _predicateFormat.replace(/\$value/g, "%@"),
+            values       = @[];
+
+        for (var i = 0; i < valueCount; i++)
+            [values addObject:aValue];
```

Comment:
Nice creative solution here 😃 

What about doing it like this?
```
var values = @[],
    formatString = _predicateFormat.replace(/\$value/g, function(x) {[values addObject:aValue]; return "%@";});             
```

Reply:
Even more creative !.. 😄
I love this one !.. (I should have found this by myself 😉)
OK, I'll adapt ASAP.